packages/api-electron/README.md

```diff
@@ -20,23 +20,15 @@ This module needs to register IPC handlers once the app `ready` event has fired.
 const ssfElectron = require('ssf-desktop-api-electron');
 
 function createWindow () {
-  ssfElectron();
+  ssfElectron(url);
   // ... other stuff!
 }
 
 app.on('ready', createWindow)
 ```
 
-### Renderer process integration
+where `url` is the url to load in the hidden window.
 
-Add the following pre-load script to any browser window that needs to use this API:
+### Renderer process integration
 
-```
-win = new BrowserWindow({
-  // ...
-  webPreferences: {
-    // ...
-    preload: path.join(__dirname, 'node_modules/ssf-desktop-api-electron/preload.js')
-  }
-})
-```
+The call to ssfElectron will create a BrowserWindow that will not be shown, and will load the url passed into `ssfElectron()`. From here, use `ssf.window()` to create windows.
```

Comment:
Maybe just _browser window_, instead of _BrowserWindow_? The latter feels like an internal implementation detail, that I don't think the user needs on know about.